docs: update docs website (#362)
Azure/secrets-store-csi-driver-provider-azure: website/content/en/configurations/identity-access-modes/user-assigned-msi-mode.md

---
type: docs
title: "User-assigned Managed Identity"
linkTitle: "User-assigned Managed Identity"
weight: 3
description: >
  Use a User-assigned Managed Identity to access Keyvault.
---

> Supported with Linux and Windows

<details>
<summary>Examples</summary>

- `SecretProviderClass`
```yaml

> # This is SecretProviderClass example using user-assigned identity for authentication with Key Vault

Reviewer: ```suggestion
# This is a SecretProviderClass example using user-assigned identity to access Key Vault

```

apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
kind: SecretProviderClass
metadata:
  name: azure-kvname-user-msi
spec:
  provider: azure
  parameters:
    usePodIdentity: "false"
    useVMManagedIdentity: "true"
    userAssignedIdentityID: "<client id of user assigned identity>"
    keyvaultName: "kvname"
    cloudName: ""                   # [OPTIONAL for Azure] if not provided, azure environment will default to AzurePublicCloud
    objects:  |
      array:
        - |
          objectName: secret1
          objectType: secret        # object types: secret, key or cert
          objectVersion: ""         # [OPTIONAL] object versions, default to latest if empty
        - |
          objectName: key1
          objectType: key
          objectVersion: ""
    tenantId: "tid"                 # the tenant ID of the KeyVault  
``` 

- `Pod` yaml
```yaml

> # This is a sample pod definition for using SecretProviderClass and user-assigned identity for authentication with Key Vault

Reviewer: ```suggestion
# This is a sample pod definition for using SecretProviderClass and user-assigned identity to access Key Vault

```
